src/client/reducers/reducer-learner.jsx

```diff
@@ -1,17 +1,25 @@
 import {
-  FETCH_LEARNERS,
   DONE_LOADING,
```

Comment:
I think that in the rebase from master, you're gonna have to add `SET_SKILLS` ... please consult @NeoJax for any questions about https://github.com/LearnersGuild/talent/pull/72/